lib/tasks/deployment/20240626145318_sha256_action_text_attachment_migrate.rake

```diff
@@ -0,0 +1,33 @@
+namespace :after_party do
+  desc 'Deployment task: sha256_action_text_attachment_migrate'
+  task sha256_action_text_attachment_migrate: :environment do
+    puts "Running deploy task 'sha256_action_text_attachment_migrate'"
+
+    # https://discuss.rubyonrails.org/t/rails-6-1-7-gives-404-for-actiontext-attachments/80803
+    def refresh_trix(trix)
+      return unless trix.embeds.size.positive?
+
+      trix.body.fragment.find_all('action-text-attachment').each do |node|
+        embed = trix.embeds.find do |attachment|
```

Comment:
I'm not familiar with the API, but: if there are multiple embeds, shouldn't be checking and processing all of them and not just the first?

Reply:
updated code to clarify this - it's trying to find the embed matching the action-text-attachment node it is processing